.github/workflows/CI.yml

```diff
@@ -117,5 +117,4 @@ jobs:
         if: runner.os != 'Linux'
       - name: Package
         run: |
```

Comment:
Can you update this from

```yml
run: |
  yarn package
```

to be
```yml
run: yarn package
```

Other than that I like this PR. Once this is updated I am willing to approve!